faq/models.py

```diff
@@ -0,0 +1,22 @@
+from django.db import models
+
+
+class QuestionCategory(models.Model):
+    name = models.CharField(max_length=50)
+
+    def __str__(self):
+        return self.name
+
+
+class Question(models.Model):
+    question = models.CharField(max_length=255)
+    answer = models.TextField(blank=True)
```

Comment:
@xavierdutreilh afaik it is actually recommended **not to use** null=True on string-based fields: https://docs.djangoproject.com/es/1.9/ref/models/fields/#django.db.models.Field.null